introducing-polyglotweekly.com.md

```diff
@@ -0,0 +1,193 @@
+<!--
+headline: Introducing polyglotweekly.com
+description: introducing the Polyglot Weekly project
+author: Ben Coe
+datePublished: 2015-04-05
+twitter: benjamincoe
+github: bcoe
+-->
+
+It's been almost a year since I wrote an article about programming, this makes me me sad.
+
+> Polyglot Weekly grows from the question, "what would get me blogging more often?"
```

Comment:
Is this a pull quote? It'd be cool to have some sort of markup within the articles allowing us to programmatically achieve this.